src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -160,22 +161,27 @@ def _get_source_file(assembly_dir: str, assembly_file: str) -> str:
     return src_file
 
 
-def _upload_assembly_to_workspace(
-    conf: Conf,
-    workspace_id: int,
-    file_path: str,
-    assembly_name: str,
-) -> str:
-    """Upload an assembly file to workspace."""
+def _upload_assemblies_to_workspace(
+        conf: Conf,
+        workspace_id: int,
+        file_paths: tuple[str],
+        assembly_names: tuple[str],
```

Comment:
My assumption is that `file_paths` and `assembly_names` get passed in is 1:1 match in tuples. Or my assumption is too strong here?

Reply:
I'm with Tian on this one, I'd pass a dict or a list of named tuples or something. Easier to read, easier to understand they're both supposed to be the same length, less likely that someone will add buggy code because they think the lists are independent

Reply:
Updated with a list of named tuples